index.html

```diff
@@ -405,8 +405,278 @@ <h4 class="title-small">Winning the talent war with a scalable tech skills platf
+                        <h6 class="footer-sub-heading-text">Authors</h6>
+                        <h6 class="footer-sub-heading-text">prefssional services</h6>
+                    </div>
+                </div>
+                <div class="footer-arrow">
+                    <img src="/aby-website/images/chevron-down.svg" alt="">
+                </div>
+            </div>
+            <div class="footer-elements">
+                <div>
+                    <h5 class="footer-heading-text">Company</h5>
+                    <div class="footer-sub-heading">
+                        <h6 class="footer-sub-heading-text">About us</h6>
+                        <h6 class="footer-sub-heading-text">Customer stories</h6>
+                        <h6 class="footer-sub-heading-text">careers</h6>
+                        <h6 class="footer-sub-heading-text">Blog</h6>
+                        <h6 class="footer-sub-heading-text">News room</h6>
+                        <h6 class="footer-sub-heading-text">Resource centre</h6>
+                    </div>
+                </div>
+                <div class="footer-arrow">
+                    <img src="/aby-website/images/chevron-down.svg" alt="">
+                </div>
+            </div>
+            <div class="footer-elements">
+                <div>
+                    <h5 class="footer-heading-text">Resources</h5>
+                    <div class="footer-sub-heading">
+                        <h6 class="footer-sub-heading-text">Download pluralsight</h6>
+                        <h6 class="footer-sub-heading-text">Events</h6>
+                        <h6 class="footer-sub-heading-text">Teach</h6>
+                        <h6 class="footer-sub-heading-text">Partners</h6>
+                        <h6 class="footer-sub-heading-text">Afiliate programs</h6>
+                        <h6 class="footer-sub-heading-text">pluralsight one.org</h6>
+                    </div>
+                </div>
+                <div class="footer-arrow">
+                    <img src="/aby-website/images/chevron-down.svg" alt="">
+                </div>
+            </div>
+            <div class="footer-elements">
+                <div>
+                    <h5 class="footer-heading-text">Contact</h5>
+                    <div class="footer-sub-heading">
+                        <h6 class="footer-sub-heading-text">contact</h6>
+                        <h6 class="footer-sub-heading-text">Help center</h6>
+                        <h6 class="footer-sub-heading-text">IP whitelist</h6>
+                        <h6 class="footer-sub-heading-text">site-map</h6>
+                    </div>
+                </div>
+                <div class="footer-arrow">
+                    <img src="/aby-website/images/chevron-down.svg" alt="">
+                </div>
+            </div>
+        </div>
+        <div class="footer-bottom-area">
+            <div class="footer-icons">
+                <div class="footer-icon-size">
```

Comment:
Why did you put size? I think just "footer-icon" is enough

Reply:
for different sizes for mobile and desktop

Reply:
and for better spacing

Reply:
I think footer-icon is a better name because size doesn't add meaning to it.

Reply:
sure done